model/src/main/kotlin/OrtResult.kt

```diff
@@ -412,13 +412,19 @@ data class OrtResult(
     fun getPackage(id: Identifier): CuratedPackage? = packages[id]?.curatedPackage
 
     @JsonIgnore
-    fun getPackages(): Set<CuratedPackage> = analyzer?.result?.packages.orEmpty()
+    fun getPackages(omitExcluded: Boolean = false): Set<CuratedPackage> =
+        analyzer?.result?.packages.orEmpty().filterTo(mutableSetOf()) { pkg ->
+            !omitExcluded || !isExcluded(pkg.pkg.id)
+        }
 
     /**
-     * Return all [Project]s contained in this [OrtResult].
+     * Return all [Project]s contained in this [OrtResult] or only the non-excluded ones if [omitExcluded] is true.
```

Comment:
While at it, can we please get docs for `getPackages()`, too?

Reply:
Done.